NodeJS/sample/SampleAUXCodeClient.ts

```diff
@@ -0,0 +1,58 @@
+import * as Constants from "../src/Constants"
+import { createAUXCodeClient, AUXCodeClient } from "../src/AUXCodeClient"
+import { getValue } from "../src/Utils"
+
+const args = require('minimist')(process.argv.slice(2));
+
+let endpoint: string
+let apiKey: string
+
+try {
+    endpoint = getValue(Constants.ENDPOINT_KEY, args)
+    apiKey = getValue(Constants.API_KEY, args)
+    main();
```

Comment:
I don't think the try-catch will work for main() as it's an async method

you should probably use :
```
main()
.catch(error => { console.log(error) } )
.finally( () => process.exit(-1) )
```